docs/clusterdefinition.md

```diff
@@ -31,7 +31,8 @@ Here are the valid values for the orchestrator types:
 |Name|Required|Description|
 |---|---|---|
 |kubernetesImageBase|no|This specifies the image of kubernetes to use for the cluster.|
-|networkPolicy|no|Specifies the network policy tool for the cluster. Default is `none`, which won't enforce network policy. This can be set to `calico` for clusters with Linux agents only.|
+|networkPolicy|no|Specifies the network policy tool for the cluster. Valid values are:<br>`none` (default), which won't enforce any network policy,<br>`azure` for applying Azure VNET network policy,<br>`calico` for Calico network policy for clusters with Linux agents only.<br>See [network policy examples](../examples/networkpolicy) for more information.|
+|clusterSubnet|no|The IP subnet used for allocating IP addresses for pod network interfaces.|

```

Comment:
Are there any restrictions on what this subnet can be? For example does it have to fall under the VNet supernet? Must it be RFC1918 compliant?